api/Navigator.json

```diff
@@ -1062,6 +1062,53 @@
           }
         }
       },
+      "getInstalledRelatedApps": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "80"
+            },
+            "chrome_android": {
+              "version_added": "80"
+            },
+            "edge": {
+              "version_added": "80"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "67"
+            },
+            "opera_android": {
+              "version_added": false
```

Comment:
I've confirmed on Opera for Android that the method isn't exposed. Curious, but OK.